W 12217812 credentialusers se (#124)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

To avoid confusion, append the name of the time zone to the name of the application or the downtime. For example, name your application `Words - Tokyo` and your downtime `Electrical Power Outage Tokyo`.

To create a process application:

. Open the *Process Applications* view of the *Process Automation* module.
. Click *Create*.

> . [[form-create-process-applications]] Complete the form *Create a new application*:

Author: changed all occurences

==== Create a Global Variable

> A global variable consists of a name, description, type, and value. Team members can link them to activity paramters in the test or the production phase. 

Reviewer: You had a mismatch between "A global variable" and "link them."

==== Create Credentials

> Create credentials with which a bot can login to an external application during process runs.

Reviewer: ```suggestion
Create credentials with which a bot can log in to an external application during process runs.
```

Use descriptive names for credentials and only use them in an appropriate context. If, for example, the employee `Nina` has an account for two applications, `Letters` and `Numbers`, create either two credentials (`nina_account_letters` and `nina_account_numbers`) or a single one that includes both applications in its name (`nina_accounts_letters_numbers`).

To create a credential:

. Open the *Credential Pool* view of the *Process Automation* module.
. Click *Create*.
. [[form-create-credential]] Complete the form *Create a New Credential*:
* [[form-create-credential-useraffiliation]] *User affiliation*:
+

> Click to choose users that may view, edit, and use the credential. Add yourself to be able to view, edit, and use the credential, too.

Reviewer: I suggested wording to make the text more active in the text above.

> Click to choose users that may view, edit, and use the credential. Add yourself to be able to view, edit, and use the credential, too.

Author: I left may, because it's about permissions.

* *User group affiliation*:
+

> Click to choose user groups whose users may view, edit, and use the credential.

Reviewer: ```suggestion
Click to choose user groups whose users can view, edit, and use the credential.
```

Author: I left 'may' because it's about permissions.

The host address used to get the credentials. The default address for Outlook services is `https://login.microsoftonline.com/common/oauth2/v2.0/`. This address might change if the customer uses a self hosted service.
*** *Client ID*:
+

> To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application has an ID, which is unique in the entire OAuth Host space, and that allows users to explicitly grant or revoke access for this Azure application to their accounts.

Reviewer: ```suggestion
To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application must have an ID that is unique in the entire OAuth Host space and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
```

Author: Every Azure application has a unique ID, I suppose. This text was provided by our developers. Unfortunately I don't have an Azure account, so I can't test this.

==== Check the Usage of a Credential

> Before editing a credential, check its usage to avoid inadvertent side-effects. You can only check the usage a credential if you belong to the <<form-create-credential-useraffiliation, affiliated users>> of that credential.

Reviewer: Maybe 
<<form-create-credential-useraffiliation, affiliated users>> of that credential > that credential's <form-create-credential-useraffiliation, affiliated users>> group
